resources/assets/js/store/modules/currentUser.js

```diff
@@ -58,6 +58,7 @@ const getters = {
 	currentUserStats: state => state.stats,
 	currentUserSubscriptionDates: state => state.profile.subscription.dates,
 	currentUserSubscriptionActive: state => state.profile.subscription.status === 'active',
+	currentUserHasLatestProduct: state => state.profile.subscription.has_latest_product,
```

Comment:
If https://github.com/bethinkpl/wnl-platform/pull/1361 is merged first, this needs to be updated to:
```suggestion
	currentUserHasLatestProduct: state => state.subscription.has_latest_product,
```